v23.1/advanced-changefeed-configuration.md

```diff
@@ -9,37 +9,140 @@ docs_area: stream_data
+
+For high durability delivery, Cockroach Labs recommends setting:
+
+~~~
+kafka_sink_config='{'RequiredAcks': 'ALL'}'
+~~~
+
+`ALL` provides the highest consistency level. A quorum of Kafka brokers that have committed the message must be reached before the leader can acknowledge the write. 
 
-- Increase the number of seconds before [garbage collection](architecture/storage-layer.html#garbage-collection) with the [`gc.ttlseconds`](configure-replication-zones.html#gc-ttlseconds) setting to provide a higher recoverability window for data if a changefeed fails. For example, if a sink is unavailable, changes are queued until the sink is available again. While the sink is unavailable, changes will be retried until the garbage collection window is reached and then the data is removed.
-  - You can also use the [`protect_data_from_gc_on_pause`](create-changefeed.html#protect-pause) option in combination with [`on_error=pause`](create-changefeed.html#on-error) to explicitly pause a changefeed on a **non**-retryable error (instead of going into a failure state) and to then protect the changes from garbage collection.
-- Determine what a successful write to Kafka is with the [`kafka_sink_config: {"RequiredAcks": "ALL"}`](changefeed-sinks.html#kafka-required-acks) option, which provides the highest consistency level.
-- Use [Kafka](changefeed-sinks.html#kafka) or [cloud storage](changefeed-sinks.html#cloud-storage-sink) sinks when tuning for high durability delivery in changefeeds. Both Kafka and cloud storage sinks offer built-in advanced protocols, whereas the [webhook sink](changefeed-sinks.html#webhook-sink), while flexible, requires an understanding of how messages are acknowledged and committed by the particular system used for the webhook in order to ensure the durability of message delivery.
-- Ensure that data is ingested downstream in its new format after a schema change by using the [`schema_change_events`](create-changefeed.html#schema-events) and [`schema_schange_policy`](create-changefeed.html#schema-policy) options. For example, setting `schema_change_events=column_changes` and `schema_change_policy=stop` will trigger an error to the `cockroach.log` file on a [schema change](changefeed-messages.html#schema-changes-with-column-backfill) and the changefeed to fail.
+{{site.data.alerts.callout_info}}
+{% include {{ page.version.version }}/cdc/kafka-acks.md %}
+{{site.data.alerts.end}}
+
+### Choosing changefeed sinks
+
+Use [Kafka](changefeed-sinks.html#kafka) or [cloud storage](changefeed-sinks.html#cloud-storage-sink) sinks when tuning for high durability delivery in changefeeds. Both Kafka and cloud storage sinks offer built-in advanced protocols, whereas the [webhook sink](changefeed-sinks.html#webhook-sink), while flexible, requires an understanding of how messages are acknowledged and committed by the particular system used for the webhook in order to ensure the durability of message delivery.
```

Comment:
Is this still true for v23.1 with the webhook performance improvements/coming out of preview?